report_thesis/src/sections/background.tex

```diff
@@ -227,28 +227,33 @@ \subsubsection{Extra Trees Regressor (ETR)}
 However, it often achieves better generalization performance, especially in high-dimensional or noisy datasets.
 
 \subsubsection{Gradient Boosting Regression (GBR)}\label{sec:gradientboost}
-In this section we introduce \gls{gbr} primarily based on \citet{James2023AnIS}.
-\gls{gbr} is an ensemble learning method that sequentially construct multiple decision trees, each one correcting the errors of the previous one, using gradient descent and boosting techniques.
-Each tree is small, with few terminal nodes, preventing large adjustments based on a single tree's predictions.
-This ensures that each tree makes small and simple error corrections, such that each step refines the model's performance more reliably.
+In this section, we introduce \gls{gbr} primarily based on \citet{hastie_elements} and \citet{burkovHundredpageMachineLearning2023}.
 
-Initially, the prediction is set as $\hat{f}_0(\mathbf{x}) = 0$ and residuals as $r_i = y_i$ for all $i$ in the training set, where $\mathbf{x}$ represents the vector of input feature(s) and $y$ is the true value or target variable.
-The model is then iteratively improved over $B$ iterations, where $B$ is a hyperparameter controlling the total number of trees.
-With each iteration $b$ from $1$ to $B$, predictions are updated as:
-$$
-    \hat{f}^{(b)}(\mathbf{x}) = \hat{f}^{(b-1)}(\mathbf{x}) + \lambda \hat{f}_b(\mathbf{x}),
-$$
-where $\hat{f}_b(\mathbf{x})$ is the prediction of the $b$-th tree and $\lambda$ is the learning rate. Residuals are updated as:
-$$
-    r_i^{(b)} = y_i - \hat{f}^{(b-1)}(\mathbf{x}_i),
-$$
-Each tree is then trained on these updated residuals.
-The repetitive process of fitting a weak learner to predict the residuals and using its predictions to update the model gives us the final model:
+Gradient Boosting is a machine learning technique used for various tasks, including regression and classification.
+The fundamental concept involves sequentially adding models to minimize a loss function, where each successive model addresses the errors of the ensemble of preceding models.
```

Comment:
Would it be worthwhile referring back to decision trees somewhere around here? Given the reworked structure of the background section it might make sense to just drive home the point.

Reply:
Update to specifically mention decision trees. I realized that I scarcely mentioned them at all. I couldn't really find a nice way to integrate a ref back to our section about it. Do you think it's fine that I don't explicitly call back to our section about them? I don't think any other section does, as that section has no label.